pkg/apis/signaturealgo/signature_digest_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package signaturealgo
+
+import (
+	"crypto"
+	"testing"
+)
+
+func TestHashAlgorithm(t *testing.T) {
```

Comment:
nit, maybe add an invalid test case. I see it tested other places, but might as well test it here to get full coverage in this unit test.